plugins/funind/recdef.ml

```diff
@@ -105,12 +105,7 @@ let next_ident_away_in_goal ids avoid =
 
 let compute_renamed_type gls id =
   let t = Tacmach.pf_get_hyp_typ id gls in
-  if Clenv.rename_with () then
-    rename_bound_vars_as_displayed (Global.env ()) (Proofview.Goal.sigma gls)
-      (*no avoid*) Id.Set.empty (*no rels*) []
-      t
-  else
-    t
+  t
```

Comment:
I think you should go one step further and replace both occurrences of `compute_renamed_type` with `Tacmach.pf_get_hyp_typ`. As it stands, the function no longer does what its name says it does.

Other than that, it looks good to me, assuming the VST failure is unrelated.